test/serializers/cache_test.rb

```diff
@@ -160,9 +161,23 @@ def test_serializer_file_path_on_windows
         assert_equal caller_line[ActiveModel::Serializer::CALLER_FILE], path
       end
 
+      def test_serializer_file_path_with_space
+        path = '/Users/git/ember js/ember-crm-backend/app/serializers/lead_serializer.rb'
+        caller_line = "#{path}:1:in `<top (required)>'"
+        assert_equal caller_line[ActiveModel::Serializer::CALLER_FILE], path
+      end
+
+      def test_serializer_file_path_with_submatch
+        # The submatch in the path ensures we're using a correctly greedy regexp.
+        path = '/Users/git/ember js/ember:123:in x/app/serializers/lead_serializer.rb'
```

Comment:
Yes, quite.

In my case the file path was generated by another process with apparently little sanitation on the naming ;-)